### ISSUE TITLE
Feature Request AT power calculator 

### ISSUE THREAD

**Issue by CIH137** [gmiclotte/gear-optimizer#39]
AT power calculator and target power with beard

maybe also power after x days

**Comment by gmiclotte** [comment]
Good idea, no promises as to when I'll have the time to implement this though.

**Comment by joshuaRMS** [comment]
Will suggested features you like be implemented if someone opens a pull request for it?

**Comment by gmiclotte** [comment]
I will continue to merge useful pull requests and update the webpage, but I won't be implementing any new features myself for this project.
